packages/core/src/domain/tracekit.spec.ts

```diff
@@ -112,7 +112,7 @@ Error: foo
       expect(computeStackTrace(2).message).toBeUndefined()
       expect(computeStackTrace({ foo: 'bar' }).message).toBeUndefined()
       expect(computeStackTrace(undefined).message).toBeUndefined()
-      // tslint:disable-next-line:no-null-keyword
+      // eslint-disable-next-line no-null/no-null
```

Comment:
The TSlint comments were converted to ESLint automatically by the [tslint-to-eslint-config](https://github.com/typescript-eslint/tslint-to-eslint-config) tool. I was going to remove the rules we agreed on not using and their comments in the next PR (since we didn't decide on all rules yet). But now, I will remove the ones we already agreed on removing and handle the rest in the next PR.